recipes/r-dotcall64/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = '0.9-04' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-dotcall64
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: dotCall64_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/dotCall64_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/dotCall64/dotCall64_{{ version }}.tar.gz
+  sha256: 0e65a56c836aecd02380d9033989a05e391552eb6b3a9d16504a827b9faae662
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  skip: True  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('dotCall64')"  # [not win]
+    - "\"%R%\" -e \"library('dotCall64')\""  # [win]
+
+about:
+  home: https://git.math.uzh.ch/reinhard.furrer/dotCall64
+  license: GPL (>= 2)
+  summary: ' An alternative version of .C() and .Fortran() supporting long vectors and 64-bit
```

Comment:
Summary is an optional entry in `conda-forge`. This should not be a blocker for the merge.